### DISCUSSION TITLE
How to further narrow DataGrid rows padding and also in between column

### DISCUSSION THREAD

**Discussion by shakeeluetian** [Megabit/Blazorise#2760] [General]
a) I have used `<DataGrid Narrow="true">` but i want to further reduce padding. How can i do that? 
b? How can i decrease the padding in between `DataGrid `columns?
c) Can we apply CSS on `<DataGridColumns>` ?

**Comment by stsrki** [comment]
It's not hard. First add the custom class name on DataGrid, to make sure you don't target the wrong tables:

```cshtml
<DataGrid Class="my-data-grid"
            @ref="dataGrid"
            TItem="Employee"
```

and then have some CSS

```css
.my-data-grid tbody tr td {
    padding: .25rem;
}
```

**Reply by shakeeluetian** [reply]
Thank u very much
